where_arg_references() got an unexpected keyword argument 'return_decorator'

https://softwarefactory-project.io/logs/30/1130/a00c60741cf9398283874636ea8ff5cd957e7da8/check/packit-service-tests-requre/4cbc9e7/job-output.txt.gz

==================================== ERRORS ====================================
_____ ERROR collecting tests_requre/openshift_integration/test_pkgtool.py ______
ests_requre/openshift_integration/test_pkgtool.py:45: in <module>
    decorate=StoreFiles.where_arg_references(
 E   TypeError: where_arg_references() got an unexpected keyword argument 'return_decorator'

--- a/tests_requre/openshift_integration/test_pkgtool.py
+++ b/tests_requre/openshift_integration/test_pkgtool.py
@@ -44,7 +44,7 @@ from packit.pkgtool import PkgTool
         what="git.repo.base.Repo.clone_from",
         decorate=StoreFiles.where_arg_references(
             key_position_params_dict={"to_path": 2},
-            return_decorator=Repo.decorator_plain,
+            output_cls=Repo,
         ),
     )
 )

